README.rst

```diff
@@ -147,6 +147,23 @@ you can make use of the ``IGNORE_WARNINGS`` flag. For example:
   >>> np.mean([])  # doctest: +IGNORE_WARNINGS
   np.nan
 
+Showing warnings
+~~~~~~~~~~~~~~~~
+
+If code in a doctest emits a warning and you want to make sure that warning is
+shown, you can make use of the ``SHOW_WARNINGS`` flag. This is useful when
+warnings are turned into errors by pytest, and also because by default warnings
+are printed to stderr. This is the opposite from ``IGNORE_WARNINGS`` so
+obviously the two flags should not be used together. For example:
+
+.. code-block:: python
+
+  >>> import numpy as np
+  >>> np.mean([])  # doctest: +SHOW_WARNINGS
```

Comment:
One more question: If the expected warning does not show, will doctest fail or ignore?

Is this the doctest equivalent of `pytest.warns`?

Reply:
> One more question: If the expected warning does not show, will doctest fail or ignore?

Yep. I added more tests.

> Is this the doctest equivalent of pytest.warns?

Yes, it can be seen as such.